runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -4140,6 +4140,32 @@ class INTERPRETER_CLASS
 
 		return rc;
 	}
+
+	VMINLINE VM_BytecodeAction
+	inlUnsafeGetObjectSize(REGISTER_ARGS_LIST)
+	{
+		j9object_t o = *(j9object_t*)_sp;
+		VM_BytecodeAction rc = EXECUTE_BYTECODE;
+
+		if (NULL == o) {
+			rc = THROW_NPE;
+		} else {
+			J9Class *oClass = J9OBJECT_CLAZZ(_currentThread, o);
+
+			if (J9CLASS_IS_ARRAY(oClass)) {
+				I_64 arrayDataSize = J9INDEXABLEOBJECT_SIZE(_currentThread, o) * J9ARRAYCLASS_GET_STRIDE(oClass);
+				I_64 headerSize = J9VMTHREAD_CONTIGUOUS_HEADER_SIZE(_currentThread);
+				returnDoubleFromINL(REGISTER_ARGS, arrayDataSize + headerSize, 2);
+			}
```

Comment:
Take a look at the coding guideline, https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md


else has to be like `} else {`

Reply:
addressed in 99c66f9